src/binary/uint.rs

```diff
@@ -42,13 +41,20 @@ impl DecodedUInt {
     }
 
     /// Interprets all of the bytes in the provided slice as big-endian unsigned integer bytes.
-    pub(crate) fn big_uint_from_slice(uint_bytes: &[u8]) -> BigUint {
-        BigUint::from_bytes_be(uint_bytes)
+    pub(crate) fn big_uint_from_slice(uint_bytes: &[u8]) -> u128 {
```

Comment:
This name might be a little misleading since it no longer returns a `BigUint`.

Reply:
Yeah, fair. It's paired with another method above (`small_uint_from_slice`) that returns a `u64`. They should be merged anyway.